### DISCUSSION TITLE
Set timestamp in the prompt?

### DISCUSSION THREAD

**Discussion by robertmarkbram** [ohmyzsh/ohmyzsh#12624] [Help] [answered]
I am using zsh 5.9 with ohmyzsh. I seem to remember my prompt having the full date on it. I re-ran `p10k configure` but that didn't give me a way to add date stamp to the prompt.

```
 ~/.oh-my-zsh  on master *2 ?1 ---------- at 09:46:07 am
>
```

There is no `PROMPT` in my .zshrc and I am having trouble figuring out how to do this. 

Rob
:)

**Comment by mcornella** [comment, marked as answer]
If you are using `p10k` you can edit the `.p10k.zsh` file in your home directory to add time prompt sequences, for which you can reference the options in https://zsh.sourceforge.io/Doc/Release/Prompt-Expansion.html#Date-and-time.

**Reply by robertmarkbram** [reply]
Thank you. I found my .p10K.zsh file had these lines:

```
  # Format for the current time: 09:51:02. See `man 3 strftime`.
  typeset -g POWERLEVEL9K_TIME_FORMAT='%D{%I:%M:%S %p}'
```

So, using `man 3 strftime` I figured out how to alter the prompt to what I wanted.
